### ISSUE TITLE
Crash Team Racing Hangs at Credits

### ISSUE THREAD

**Issue by Monsterray** [Mystro256/wiisxr#40]
After I start Crash Team Racing the game shows the Sony Computer Entertainment Credits at the beginning but gets stuck there and plays all the dialogue of the game one after another. This is where it gets stuck at.

![hang at beginning](https://cloud.githubusercontent.com/assets/1779600/19839698/d4666fe4-9ea3-11e6-8823-d28d0b7acc21.jpg)

**Comment by leodexe** [comment]
This game is well known to have issues with most emulators, not only WiiSXR. It is related with the SPUsync function, that handles some sort of audio streaming that most emulators can't deal with properly. But in the case of WiiSXR, THERE IS one version that actually WORKS, although it has some graphical issues (like crates and item boxes being invisible most of the time). It is the SCUS-94459. The other versions WILL NOT WORK. Because I tested them myself. You can find more info here in this video. Although it is in Spanish, it provides useful information regarding playing CTR on WiiSXR.
Video here -> https://www.youtube.com/watch?v=jk3C43UwBfc

**Late 2022 edit:** Although SCUS-94459 is a demo according to many people and many websites, this specific one where the youtuber talks in the video states that it is not the regular demo, but rather a modified version of the demo, which includes everything, all characters and all tracks, it is not any ordinary demo due to the fact SCUS-94459 wasn't mean to be mass distributed as isn't the final game.

For **unknown reasons**, this demo and the modified demo will work in WiiSXR, but the other ones just crashes.

I don't think this is true anymore because niuus continued the work and eventually forked into **WiiSXRX** to continue what Mystro256 left behind, this new version should run ANY version of CTR and not only the demo, and it also has multitap support, which is confirmed to work with Crash Bash.

**Comment by thiagoralves** [comment]
The SCUS-94459 is reported as Demo on all ISO websites I could find. Is there a full version of the game that is compatible?

**Comment by leodexe** [comment]
You are right. In the youtube video URL I left in my comment there is a link in the description with the full version of the game. If you understand Spanish you should watch the video too.

**Comment by BellezaEmporium** [comment]
Yes you are right @Mystro256 , this was a duplicate. Did not know that this issue already existed.

SCUS94459 is indeed a exclusive demo disc, please check at *REMOVED* for a disc image.
Wiki should have been changed to fix this little typo.

SPUsync is actually not taking CTR's XA files well.

**Comment by Mystro256** [comment]
@ShadixAced Please refrain from mentioning locations to acquire pirated games.

**Comment by BellezaEmporium** [comment]
Well it was only the website that hosted the disc IMAGE, do not know about if they actually distribute the game.
